django_ipam/tests/base/test_commands.py

```diff
@@ -30,3 +30,10 @@ def test_import_subnet_command(self):
             call_command('import_subnet', file='invalid.pdf')
         with self.assertRaises(CommandError):
             call_command('import_subnet', file='invalid_path.csv')
+
+    def tearDownClass():
+        import glob
+        import os
```

Comment:
mm.. please place these imports at the top of the file

Reply:
Is it a bad practice to do so? I have seen people doing it . Will try avoiding it from next time if its not a good practice

Reply:
it's bad unless you have a very specific need for doing it, which usually boils down to avoiding circular imports, and even in these cases, it should be used as a last resort